shared/wallets/wallet-list/container.js

```diff
@@ -0,0 +1,44 @@
+// @flow
+import {WalletList} from '.'
+import logger from '../../logger'
+import {connect, type TypedState, type Dispatch} from '../../util/container'
+import {getAccounts} from '../../constants/wallets'
+
+const mapStateToProps = (state: TypedState) => ({accounts: getAccounts(state)})
+
+const mapDispatchToProps = (dispatch: Dispatch, {navigateUp}) => ({
+  onAddNew: () => {
+    logger.error('TODO: onAddNew')
+  },
+  onLinkExisting: () => {
+    logger.error('TODO: onLinkExisting')
+  },
+})
+
+const mergeProps = (stateProps, dispatchProps) => {
+  // Accounts is already ordered, so no need to sort.
+  const wallets = stateProps.accounts
+    .map(a => {
+      let name = a.name || a.accountID
+      // TODO: Better way to do this?
+      if (name.length > 20) {
+        name = name.substr(0, 20) + '...'
+      }
+
+      return {
+        // TODO: How to get keybaseUser?
```

Comment:
We can't just assume that every wallet should have the current user's avatar, right?

Reply:
Only the default has the avatar. You can use config.username

Reply:
Fixed!